### DISCUSSION TITLE
Is there a way to configure vscode-java to auto download maven dependencies src and javadoc?

### DISCUSSION THREAD

**Discussion by rafaeltuelho** [redhat-developer/vscode-java#2707] [Q&A] [answered]
I just posted this Q on Stackoverflow before I knew this Q&A on Github.

https://stackoverflow.com/questions/73894704/is-there-a-way-to-configure-vscode-java-to-auto-download-maven-dependencies-src

**Comment by snjeza** [comment, marked as answer]
@rafaeltuelho  you can try to add
```
"java.maven.downloadSources": true,
```
to your settings.json

**Reply by rafaeltuelho** [reply]
Cool!
Will try that seeting. Thanks
